File: Estrutura_Repetição/Python/Ex1099.py
n = int(input())

for i in range(0, n):
    soma: int = 0
    x, y = map(int, input().split())
    menor = min(x, y)
    maior = max(x, y)
    for j in range(menor + 1, maior):
        if j % 2 != 0:
            soma += j
    print(soma)

File: Estrutura_Repetição/JavaScript/Ex1099.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { 
    return lines.shift();
};

    var n = parseInt(prompt());

    for (var i = 0; i < n; i++) {
        var soma = 0;
        var entrada = prompt();
        var numeros = entrada.split(" ");
        var x = parseInt(numeros[0]);
        var y = parseInt(numeros[1]);

        var menor = Math.min(x, y);
        var maior = Math.max(x, y);

        for (var j = menor + 1; j < maior; j++) {
            if (j % 2 !== 0) {
                soma += j;
            }
        }
        console.log(soma);
    }